yogstation/code/modules/guardian/standarrow.dm

```diff
@@ -172,6 +172,7 @@
 		log_game("[key_name(H)] has summoned [key_name(G)], a holoparasite, via the stand arrow.")
 		to_chat(H, span_holoparasite("<font color=\"[G.namedatum.color]\"><b>[G.real_name]</b></font> has been summoned!"))
 		add_verb(H, list(/mob/living/proc/guardian_comm, /mob/living/proc/guardian_recall, /mob/living/proc/guardian_reset))
+		H.update_sight()
```

Comment:
did you mean `G.update_sight()`?

Reply:
I’m intending to update the host’s sight, because that’s what links up the guardian’s vision. I believe H is the host here.